DAO/project/contracts/DAO-contract/tests/utils/setup.rs

```diff
@@ -32,16 +34,41 @@ pub(crate) fn proposal_transaction(asset_id: ContractId) -> Proposal {
 }
 
 pub(crate) async fn setup() -> (ContractId, ContractId, Metadata, Metadata, u64) {
-    let (mut wallets, mut asset_ids) = setup_wallets().await;
+    let mut rng = rand::thread_rng();
+    let num_coins = 1;
+    let coin_amount = 1_000_000;
+
+    let base_asset = AssetConfig {
+        id: BASE_ASSET_ID,
+        num_coins,
+        coin_amount,
+    };
+    let mut gov_token_id = AssetId::zeroed();
+    gov_token_id.try_fill(&mut rng).unwrap();
+    let gov_token = AssetConfig {
+        id: gov_token_id,
+        num_coins,
+        coin_amount,
+    };
+    let mut other_token_id = AssetId::zeroed();
+    other_token_id.try_fill(&mut rng).unwrap();
+    let other_token = AssetConfig {
+        id: other_token_id,
+        num_coins,
+        coin_amount,
+    };
+    let assets = vec![base_asset, gov_token, other_token];
+
+    let num_wallets = 2;
```

Comment:
I would hoist this to the top where the other vars are declared and rename to not use abbreviations